ReactNative/ReactTwitter/android/app/src/main/res/drawable/buttonbackground2.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+  <item android:state_enabled="false"
+    android:drawable="@drawable/button_background_disabled2" />
```

Comment:
there is an extra asset in this PR that is not used: button-background-normal1.png
We can delete it